src/constants/localStorage.ts

```diff
@@ -34,9 +34,6 @@ export enum LocalStorageKey {
   // Discoverability
   HasSeenElectionBannerTRUMPWIN = 'dydx.HasSeenElectionBannerTRUMPWIN',
   HasSeenTradeFormMessageTRUMPWIN = 'dydx.HasSeenTradeFormMessageTRUMPWIN',
-
-  // Informational
```

Comment:
Add a migration to delete this value for users? Or add a TODO and I'll delete a bunch in a batch later~~

Reply:
I don't think this was ever set by users yet

Reply:
Actually it might due to how `useLocalStorage` works. will add a todo